carrier-io/carrier-io.github.io
# Update Carrier documentation and theme enhancement

This pull request introduces updates to the Carrier documentation, new installation guides, updated video resources, and extended documentation capabilities using jekyll-theme-chirpy. The key changes are:

- Updated documentation with recent information, available for preview at: https://karenflorykian.github.io/carrier-io.github.io/#about
- Added a detailed installation guide for the Carrier platform, including prerequisites, which can be found here: https://karenflorykian.github.io/carrier-io.github.io/posts/carrier-install/#prerequisites
- Updated video resources to keep them up-to-date with current functionality.
- Enhanced documentation capabilities by incorporating the jekyll-theme-chirpy for easier content management and improved user experience.
 

### How to Test
1. Pull the branch locally and ensure all dependencies are installed.
2. Build the Jekyll site locally and verify that the recent information and installation guides are correctly updated.
$ bash tools/init
$ bundle
$ bundle exec jekyll s
4. Check the updated video resources and confirm they work as expected.

additional info: https://chirpy.cotes.page/posts/getting-started/
